src/serverdlg.cpp

```diff
@@ -330,7 +330,7 @@ lvwClients->setMinimumHeight ( 140 );
     pbtNewRecording->setAutoDefault ( false );
     pbtRecordingDir->setAutoDefault ( false );
     edtRecordingDir->setText( pServer->GetRecordingDir() );
-    tbtClearRecordingDir->setText ( "(X)" ); // someone make me a nice icon?
+    tbtClearRecordingDir->setText(QString::fromUtf8("\u232B"));
```

Comment:
![image](https://user-images.githubusercontent.com/1549463/86517629-5ad07d80-be22-11ea-83fc-c87545a38585.png)
`<[X]` Hm.  I'd have left the comment... :D  Better than the (X), though.